web/packages/teleport/package.json

```diff
@@ -21,17 +21,17 @@
     "@opentelemetry/api": "1.9.0",
     "@opentelemetry/context-zone": "1.30.1",
     "@opentelemetry/core": "^1.30.1",
-    "@opentelemetry/exporter-trace-otlp-http": "0.57.2",
-    "@opentelemetry/instrumentation": "0.57.2",
+    "@opentelemetry/exporter-trace-otlp-http": "0.200.0",
+    "@opentelemetry/instrumentation": "0.200.0",
     "@opentelemetry/instrumentation-document-load": "0.45.0",
-    "@opentelemetry/instrumentation-fetch": "0.57.2",
+    "@opentelemetry/instrumentation-fetch": "0.200.0",
     "@opentelemetry/instrumentation-user-interaction": "0.45.0",
-    "@opentelemetry/instrumentation-xml-http-request": "0.57.2",
+    "@opentelemetry/instrumentation-xml-http-request": "0.200.0",
```

Comment:
These likely need to be rolled back because we still haven't updated opentelemetry deps. https://github.com/gravitational/teleport/issues/50687